capstone-ui/src/Components/PresidentStatePage/PresidentStatePage.jsx

```diff
@@ -21,7 +21,12 @@ export default function PresidentialStatePage( { selectedCandidates, setSelected
                 />
             </div>
             <div className="candidate-comparison-container">
-                Candidate Comparison
+                {selectedCandidates ? (selectedCandidates.map((candidate) => (
```

Comment:
Some of this code seems duplicated with code in `PresidentialRaces`. I think one way to avoid this is my previous comment to have one page for country/state view.

Reply:
I agree, that's one of the first things I plan to do now that the core things are done.